scripts/cangen/cangen/traffic.py

```diff
@@ -0,0 +1,48 @@
+"""
+Author: Manush Patel
+Date: 2024-11-24
+https://macformula.github.io/racecar/firmware/can-traffic/
+"""
+
+from dataclasses import dataclass
+
+
+@dataclass
+class CANMessage:
+    data_length: int
+    frequency: int
+
+    @property
+    def bit_count(self):
```

Comment:
you can simplify this function to 

`return 44 + 8 * self.data_length`

The extra variables don't really help with readability. It would help if you included the entire list of CAN components (like RTR, CONTROL, CRC, etc). I'm not suggesting you do that, but it is already clear from the equation which part is fixed and which is variable